pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoaderMain.java

```diff
@@ -104,6 +105,7 @@ public static void main(String[] args) throws Exception {
         options.addOption("gb", true, "google bucket file");
         options.addOption("p", false, "preprocess");
         options.addOption("pf", true, "preprocess file name");
+        options.addOption("t", true, "Auth0 tenant id");
```

Comment:
Not sure if a tenant id or a auth0 domain should be used here.

Reply:
I vote `tenant`

Reply:
Can we use the auth0 domain/tenant **string**? I don't think it's developer-friendly to have to give the **numeric** id and that id might change across databases and environments.

Reply:
@yufengwng, not quite sure what the tenant **string** is. Could you please give an example?